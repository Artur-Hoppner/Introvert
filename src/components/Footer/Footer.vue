<template>
  <div>
    <v-footer app="app">
      <v-flex class="text-xs-center">Â© 2018. All rights reserved.</v-flex>
    </v-footer>
  </div>
</template>

<script>
export default {
  name: 'Footer'
};
</script>

<style lang="scss"></style>
