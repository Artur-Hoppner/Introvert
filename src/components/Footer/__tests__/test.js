// Utilities
import { mount, createLocalVue } from '@vue/test-utils';
// Libraries
import Vue from 'vue';
import Vuex from 'vuex';
import VueRouter from 'vue-router';
import Vuetify from 'vuetify';
// Components
import Footer from '@/components/Footer/Footer.vue';

const localVue = createLocalVue();
localVue.use(Vuex, VueRouter);

describe('Footer component', () => {
  const store = new Vuex.Store({
    getters: {
      statusMessage: state => state.statusMessage
    }
  });
  let vuetify;
  beforeEach(() => {
    Vue.use(Vuetify);
    vuetify = new Vuetify();
  });
  test('test if Foter is rendered', () => {
    // Arrange;
    const wrapper = mount(Footer, {
      localVue,
      vuetify,
      store
    });
    // Assert
    expect(wrapper.element).toMatchSnapshot();
  });
});
