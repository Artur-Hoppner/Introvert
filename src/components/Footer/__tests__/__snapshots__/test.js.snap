// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component test if Foter is rendered 1`] = `
<footer
  class="v-footer v-sheet theme--light v-footer--fixed"
  style="left: 0px; right: 0px; bottom: 0px;"
/>
`;
